src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/ObjectResultExecutor.cs

```diff
@@ -27,29 +21,32 @@ public class ObjectResultExecutor : IActionResultExecutor<ObjectResult>
         /// <summary>
         /// Creates a new <see cref="ObjectResultExecutor"/>.
         /// </summary>
-        /// <param name="options">An accessor to <see cref="MvcOptions"/>.</param>
+        /// <param name="formatterSelector">The <see cref="OutputFormatterSelector"/>.</param>
         /// <param name="writerFactory">The <see cref="IHttpResponseStreamWriterFactory"/>.</param>
         /// <param name="loggerFactory">The <see cref="ILoggerFactory"/>.</param>
         public ObjectResultExecutor(
-            IOptions<MvcOptions> options,
+            OutputFormatterSelector formatterSelector,
```

Comment:
What's our level of concern for breaking change here? Particularly since this is in the .Infrastructure namespace?

Reply:
This type was pubternal in 2.0 so there's no breaking change